README.md

```
@@ -49,366 +23,98 @@ brew install malt
+| `malt create` | Generate service configs in `malt/conf/` |
+| `malt start` | Start all services |
+| `malt stop` | Stop all services |
+| `malt kill` | Force-kill all service processes |
+| `malt status` | Show running/stopped state per port |
+| `malt env` | Output shell env setup script |
+| `malt info` | Show project information |
 
-Malt creates the following directory structure in your project:
+## Project Structure
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)